src/constants/content-report.ts

```diff
@@ -0,0 +1,14 @@
+export const DMCA = 'dmca'
+export const MATURE = 'mature'
+export const OTHER = 'other'
+
+export const reasons = [DMCA, MATURE, OTHER] as const
+
+export const SENT = 'sent'
+export const FAILED = 'failed'
+export const WIP = 'wip'
+
+export const statuses = [SENT, FAILED, WIP] as const
```

Comment:
Can we either:
a. add exported types for `reason` and `status` here, similar to the `ErrorCode` type in `errors.ts` below, and use them for prop validation
or 
b. create a util function that can take a string value, and validate it against a `const` array so that TS doesn't throw an error. Now, if you try to use `reasons.includes(somestringvalue)` when validating with TS, you would get an error.

Reply:
Good call, updated in 4b0b9ee.